scalableminds/webknossos
# Refactor backend: Merge DataSetSQL and DataSet. Remove NDStore residue

### URL of deployed dev instance (used for testing):
- https://refactorbackenddataset.webknossos.xyz

### Steps to test:
- upload dataset, import it
- edit dataset properties, see if saved
- load data
- create explorative
- create task, trace some, download NML

### Issues:
- contributes to #2521 

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
